root/statistics/Index.js

```diff
@@ -0,0 +1,1078 @@
+            <th>{l_statistics('Male:')}</th>
+            <td>{formatCount(stats.data['count.artist.gender.male'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.artist.gender.male'] / nonGroupCount, 1, $c)}</td>
+          </tr>
+          <tr>
+            <th />
+            <th>{l_statistics('Female:')}</th>
+            <td>{formatCount(stats.data['count.artist.gender.female'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.artist.gender.female'] / nonGroupCount, 1, $c)}</td>
+          </tr>
+          <tr>
+            <th />
+            <th>{l_statistics('Other gender:')}</th>
+            <td>{formatCount(stats.data['count.artist.gender.other'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.artist.gender.other'] / nonGroupCount, 1, $c)}</td>
+          </tr>
+          <tr>
+            <th />
+            <th>{l_statistics('Gender not applicable:')}</th>
+            <td>{formatCount(stats.data['count.artist.gender.not_applicable'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.artist.gender.not_applicable'] / nonGroupCount, 1, $c)}</td>
+          </tr>
+          <tr>
+            <th />
+            <th>{l_statistics('with no gender set:')}</th>
+            <td>{formatCount(stats.data['count.artist.gender.null'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.artist.gender.null'] / nonGroupCount, 1, $c)}</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <h2>{l_statistics('Releases, Data Quality, and Disc IDs')}</h2>
+      <table className="database-statistics">
+        <tbody>
+          <tr className="thead">
+            <th colSpan="5">{l_statistics('Releases')}</th>
+          </tr>
+          <tr>
+            <th colSpan="3">{l_statistics('Releases:')}</th>
+            <td>{formatCount(stats.data['count.release'], $c)}</td>
+            <td />
+          </tr>
+          <tr>
+            <th />
+            <th colSpan="2">{l_statistics('by various artists:')}</th>
+            <td>{formatCount(stats.data['count.release.various'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.release.various'] / stats.data['count.release'], 1, $c)}</td>
+          </tr>
+          <tr>
+            <th />
+            <th colSpan="2">{l_statistics('by a single artist:')}</th>
+            <td>{formatCount(stats.data['count.release.nonvarious'], $c)}</td>
+            <td>{formatPercentage(stats.data['count.release.nonvarious'] / stats.data['count.release'], 1, $c)}</td>
+          </tr>
+        </tbody>
+        <tbody>
+          <tr className="thead">
+            <th colSpan="5">{l_statistics('Release Status')}</th>
+          </tr>
+          <tr>
+            <th colSpan="3">{l_statistics('Releases:')}</th>
+            <td>{formatCount(stats.data['count.release'], $c)}</td>
+            <td />
+          </tr>
+          {((Object.values(statuses): any): $ReadOnlyArray<ReleaseStatusT>).map(status => (
```

Comment:
These "any" are really not something that eslint is happy with. How could they be replaced? (these came from the original spellew work, and I'm honestly not fully sure what they do).